app-virtualization/virglrenderer/autobuild/defines

```diff
@@ -16,14 +16,15 @@ PKGBREAK="qemu<=5.1.0"
 # 7887 |       gr->size += gbm_bo_get_plane_size(bo, plane);
 #      |                   ^~~~~~~~~~~~~~~~~~~~~
 #      |                   gbm_bo_get_plane_count
-MESON_AFTER="-Dplatforms=auto \
-             -Dminigbm_allocation=false \
-             -Dvenus-validate=false \
-             -Dcheck-gl-errors=true \
-             -Ddrm-msm-experimental=false \
-             -Drender-server=false \
-             -Drender-server-worker=process \
-             -Dvideo=true \
-             -Dfuzzer=false \
-             -Dvalgrind=false \
-             -Dtracing=none"
+MESON_AFTER=(
+	"-Dplatforms=auto"
+	"-Dminigbm_allocation=true"
```

Comment:
Broken SPICE screen with `minigbm_allocation` enabled on my computer (AMD 7840HS with Radeon 780M)

This option is now deprecated in the main branch of virglrenderer (though not in the 1.2.0 release)

Reply:
Okay, will disable with a FIXME.